CSGOSimple/main.cpp

```diff
@@ -83,7 +83,12 @@ BOOL WINAPI OnDllDetach()
     return TRUE;
 }
 
-BOOL WINAPI DllMain(_In_      HINSTANCE hinstDll, _In_      DWORD     fdwReason, _In_opt_  LPVOID    lpvReserved) {
+BOOL WINAPI DllMain(	
```

Comment:
I'm not the original dev of this project & I don't have any influence but I don't see the value in changing this.

Reply:
I changed it back to original but it accidently left spaces